.changeset/eighty-teachers-itch.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
The changeset is appreciated, but not used for our `/site` projects (documentation, etc). Can you go ahead and remove this so it doesn't cause issues.